templates/documentation/vod/clip-a-video.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "clip and trim videos with api.video"
+slug: "clip-a-video"
+meta:
+  description: Clip and trim a video while uploading to api.video
+---
+
+# Video clipping & Trimming
+
+In some cases, you just need to upload a smaller piece of a video, but you would like to do that programmatically, without resorting to editing programs.
+
+[api.video](http://api.video) allows you to clip a video to upload only specific parts of the video. You will be able to specify the start and end time of the clip before you upload the source video to api.video.
+
+{% include "_partials/dark-light-image.md" dark: "/_assets/vod/video-clip/video-clipping-dark.svg", light: "/_assets/vod/video-clip/video-clipping-light.svg", alt: "A diagram that shows the process of creating and applying a watermark to a video object" %}
```

Comment:
This `alt` is for watermarks, let's use:

"A diagram that shows the process of creating a video clip"